sympy/physics/vector/functions.py

```diff
@@ -638,3 +639,81 @@ def dynamicsymbols(names, level=0,**assumptions):
 
 dynamicsymbols._t = Symbol('t')  # type: ignore
 dynamicsymbols._str = '\''  # type: ignore
+
+
+def solve_vector(f1, f2, *symbols):
```

Comment:
Shouldn't this be `solve_vector(sequence_of_vector_expressions, *symbols)` to match `solve`?